### DISCUSSION TITLE
反向代理显示connection refused

### DISCUSSION THREAD

**Discussion by SherkeyXD** [MountainDash/nonebot-bison#277] [Q&A]
配置 `BISON_OUTER_URL=http://127.0.0.1:30001/bison/`，同时利用 caddy 做反向代理，发现后台网页空白，查看 caddy 日志发现出现下面信息 `dial tcp 127.0.0.1:30001: connect: connection refused`

Caddyfile:
```
:80

reverse_proxy 127.0.0.1:30001
```

**Comment by felinae98** [comment]
可能你需要完整地描述下你的使用场景和整个网络拓补
